src/sbin/sympa.pl.in

```diff
@@ -884,12 +884,20 @@ if ($main::options{'dump'} or $main::options{'dump_users'}) {
         exit 1;
     }
 
+    my $result = $list->sync_include();
+
+    if ($result == 0) {
+        printf STDERR "No data sources defined for inclusion into list %s.\n",
+            $list->get_id;
+        exit 1;
+    }
```

Comment:
Would be better to use `$list->has_include_data_sources`.

Reply:
Adjusted.